tests/recon/torch/modules/test_latent.py

```diff
@@ -0,0 +1,60 @@
+import torch
+import unittest
+from abc import ABC, abstractmethod
+from typing import Iterator
+import torch.nn as nn
+from bdpy.recon.torch.modules import latent as latent_module
+
+
+class DummyLatent(latent_module.BaseLatent):
+    def __init__(self):
+        self.latent = torch.tensor([1.0])
+
+    def reset_states(self):
+        self.latent = torch.zeros_like(self.latent)
+    
+    def parameters(self, recurse):
+        return iter([nn.Parameter(self.latent)])
+    
+    def generate(self):
+        return self.latent
+        
+class TestBaseLatent(unittest.TestCase):
+    """Tests for bdpy.recon.torch.modules.latent.BaseLatent."""
+    def setUp(self):
+        self.latent = torch.tensor([1.0])
```

Comment:
- `expected` みたいな名前があったほうが可読性が高そうです
- `Latent` クラスのインスタンスなのか `torch.Tensor` のインスタンスなのか命名からわかりたいです

```suggestion
        self.latent_value_expected = torch.tensor([1.0])
```